on_device_training/mobilevit/README.md

```diff
@@ -0,0 +1,47 @@
+# MobileVIT End-to-End Example for On-Device Training
+This folder contains an end-to-end example for on-device training in a C# console app using the ONNXRuntime Training libraries.
+
+On-device training contains two steps: an "offline" pre-processing step that happens off of the device in question, and the training step that happens on the device. 
+
+## About the task
+The task for this example is facial expression recognition, categorizing an image into one of the following emotions:
+1. neutral
+2. happy
+3. sad
+4. surprise
+5. fear
+6. disgust
+7. angry
+
+The huggingface MobileVIT xx-small model is fine-tuned on the huggingface DiffusionFER dataset to perform this classification.
+
+## Contents
+The `mobilevit_offline.ipynb` Python notebook creates the necessary training artifacts for training. The following generated files & folders should be **migrated or moved** from the "offline" device to the device to be used for training:
+1. The training ONNX model (default name will be `training_model.onnx`)
```

Comment:
Change "The `mobilevit_offline.ipynb` Python notebook creates the necessary training artifacts for training. The following generated files & folders should be **migrated or moved** from the "offline" device to the device to be used for training:" to something like
> The `mobilevit_offline.ipynb` python notebook generates the following training artifacts: list the artifacts. 
These artifacts are necessary for training the model and should be migrated or moved to the device."